buzzfeedbotOS.py

```diff
@@ -72,33 +53,34 @@ def article_info(date, error):
 	soup = BeautifulSoup(daily_archive.content, 'html.parser')
 	for article_to_open in soup.find_all('li', attrs={'class': 'bf_dom'}):
 		#End script if post was already made longer than one hour ago
-		post_made = post_made_check(article_to_open.text)
-		if post_made == True:
-			if error == True:
+		if post_made_check(article_to_open.text):
+			if error:
 				continue
-			elif error == False:
+			else:
 				break
 		lc_art_title = article_to_open.text.lower()
+		
 		try:
-			if (article_to_open.text[0].isdigit() or article_to_open.text.lower().startswith('top')) \
-			and detect(article_to_open.text) == 'en' and not any(words in lc_art_title for words in break_words):
-	
-				no_of_points = [int(s) for s in article_to_open.text.split() if s.isdigit()] #Records number of points in the article 
-				for link in article_to_open.find_all('a', href=True):
-					top_x_link = 'https://www.buzzfeed.com' + link['href']
-					try: #Avoids rare case of when there is an index error (occurs when article starts with number immediately followed by a symbol)
-						article_text_to_use = clickbait_meat(top_x_link, no_of_points[0])
-						if article_text_to_use == '':
-							break
-						else:
-							reddit_bot(article_to_open.text, article_text_to_use, top_x_link)
-							print(article_to_open.text)
-							time.sleep(1)
-							break
-					except IndexError:
-						break
+			is_english = detect(article_to_open.text) == 'en'
 		except lang_detect_exception.LangDetectException:
 			continue
+		is_listicle = lc_art_title[0].isdigit() or lc_art_title.startswith('top')
+		has_banned_keyword = any(words in lc_art_title for words in break_words)
+		
+		if is_listicle and is_english and not has_banned_keyword:

```

Comment:
When an if's condition is too complex, sometimes it's useful to break into simpler variables. Avoids very long lines and makes the code self-documenting.